package.json

```diff
@@ -14,33 +16,43 @@
 		}
 	},
 	"scripts": {
```

Comment:
Here I've ordered scripts alphabetically.

Reply:
And added:
```
"lint-js": "wp-scripts lint-js",
"lint-js:fix": "npm run lint-js -- --fix",
```

Reply:
Yes, got that... It's good to be sorted 👍